pallets/types/src/test_utils.rs

```diff
@@ -147,3 +149,57 @@ impl BlockWithProofs {
             .collect()
     }
 }
+
+pub struct BridgeAssetLockerImpl<T>(PhantomData<T>);
```

Comment:
It is used in mocks only, what about moving it to mocks?

Reply:
Also it's used in ERC20App, EthApp and MigrationApp